src/NuGet.Core/NuGet.Common/PathUtil/PathResolver.cs

```diff
@@ -23,9 +23,9 @@ public static class PathResolver
             Func<T, string> getPath,
             IEnumerable<string> wildcards)
         {
-            IEnumerable<Regex> filters = wildcards.Select(WildcardToRegex);
+            IEnumerable<Regex> filters = wildcards.Select(WildcardToRegex).ToList();
```

Comment:
Why don't just make it `List<Regex> filters = wildcards.Select(WildcardToRegex).ToList();` for better readability?

Reply:
Fixed